src/schema/v2/FeaturedFairs/featuredFairsConnection.ts

```diff
@@ -0,0 +1,80 @@
+import { GraphQLBoolean, GraphQLFieldConfig, GraphQLNonNull } from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { filter, pick } from "lodash"
+import moment from "moment"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { fairConnection } from "../fair"
+import { createPageCursors } from "../fields/pagination"
+
+export const FeaturedFairsConnection: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  description:
+    "A connection of featured, currently running fairs, backfilled with past fairs. Fairs are sorted by start date in descending order.",
+  type: fairConnection.connectionType,
+  args: pageable({
+    includeBackfill: { type: new GraphQLNonNull(GraphQLBoolean) },
+  }),
+  resolve: async (_root, args, { fairsLoader }) => {
+    const { size, offset, page } = convertConnectionArgsToGravityArgs(args)
+
+    const numberOfRairsToFetch = size + offset
+
+    // Check for all fairs that are currently running
+    const gravityOptions = {
+      has_full_feature: true,
+      sort: "-start_at",
+      active: true,
+    }
+
+    const now = moment.utc()
+
+    const { body: unfilteredRunningFairs } = await fairsLoader(gravityOptions)
```

Comment:
This will return a default of 10 records from the gravity endpoint (as no pagination arguments are specified).